addons/sale/static/src/js/combo_configurator_dialog/combo_configurator_dialog.js

```diff
@@ -70,8 +70,8 @@ export class ComboConfiguratorDialog extends Component {
                 pricelistId: this.props.pricelist_id,
                 currencyId: this.props.currency_id,
                 soDate: this.props.date,
-                edit: true, // Hide the optional products, if any.
-                options: { canChangeVariant: false, showQuantityAndPrice: false },
+                edit: true, // TODO(loti): this "disables" optional products. Rename variable for clarity?
```

Comment:
Could you keep the comment?
```suggestion
                edit: true, // Hide the optional products, if any.
```

Reply:
I accidentally removed that. 😢
Added